src/biokbase/installed_clients/baseclient.py

```diff
@@ -4,74 +4,63 @@
 # any changes made here will be overwritten
 #
 ############################################################
-
-from __future__ import print_function
-
-import json as _json
-import os as _os
-import random as _random
-
-import requests as _requests
-
-try:
-    from configparser import ConfigParser as _ConfigParser  # py 3
-except ImportError:
-    from ConfigParser import ConfigParser as _ConfigParser  # py 2
-
-try:
-    from urllib.parse import urlparse as _urlparse  # py3
-except ImportError:
-    from urlparse import urlparse as _urlparse  # py2
-
+import json
```

Comment:
this is actually from a freshly-generated KBase sdk app with some typing stuff added.